swarm/chunk/chunk.go

```diff
@@ -138,6 +164,19 @@ const (
 // ModeSet enumerates different Setter modes.
 type ModeSet int
 
+func (m ModeSet) String() string {
+	switch m {
+	case ModeSetAccess:
+		return "Access"
+	case ModeSetSync:
+		return "Sync"
+	case ModeSetRemove:
+		return "Remove"
+	default:
+		return "Unknown Mode Set"
```

Comment:
This would be used as part of a metrics name, right? Just in case we fuck up somewhere, probably best not to have spaces in metrics names.

Reply:
Fixed.